buildSrc/src/main/java/com/google/firebase/gradle/bomgenerator/BomGeneratorTask.java

```diff
@@ -105,7 +105,6 @@ public abstract class BomGeneratorTask extends DefaultTask {
           "firebase-analytics-license",
           "firebase-annotations",
           "firebase-appcheck-interop",
-          "firebase-appcheck-safetynet",
```

Comment:
this list is based on the sdks available in maven, not in the repo. the old versions of `safetynet` are, and will be, available in maven.

Reply:
fixed